src/noisepy/seis/S1_fft_cc_MPI.py

```diff
@@ -195,14 +195,15 @@ def cross_correlate(raw_store: RawDataStore, fft_params: ConfigParameters, cc_st
                     sfft1[bb, :], sfft2[bb, :], fft_params, Nfft, fft_time[iiR][bb]
                 )
                 t3 = time.time()
-
+                print(src_chan)
```

Comment:
@mdenolle Can you remove this print or change it to use the `logger` if its needed?